axelor-business-project/src/main/java/com/axelor/apps/businessproject/service/analytic/AnalyticLineModelProjectService.java

```diff
@@ -0,0 +1,21 @@
+package com.axelor.apps.businessproject.service.analytic;
+
+import com.axelor.apps.base.AxelorException;
+import com.axelor.apps.base.db.Company;
+import com.axelor.apps.businessproject.model.AnalyticLineProjectModel;
+import com.axelor.apps.supplychain.model.AnalyticLineModel;
+
+public interface AnalyticLineModelProjectService {
+
+  boolean analyzeAnalyticLineModel(
+      AnalyticLineProjectModel analyticLineProjectModel, Company company) throws AxelorException;
+
+  AnalyticLineModel createAnalyticDistributionWithTemplate(
+      AnalyticLineProjectModel analyticLineProjectModel) throws AxelorException;
+
+  AnalyticLineProjectModel getAndCopyAnalyticDistribution(
```

Comment:
After seing the usage of this method, couldn't be possible to just use a Project as parameter instead of AnalyticLineProjectModel ?

Reply:
I don't think so as it is because it uses method that take AnalyticLineModel in param